src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.controller;
+
+import javax.validation.Valid;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
what if we want to add login method?
 it is better to add mapping for this controller (/auth)

Reply:
-